gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -638,7 +639,7 @@ void expectGeneratedMedicationStatementBasedOnReferencesTheGeneratedPlan() {
         }
 
         @Test
-        void expectGeneratedMedicationStatementEffectivePeriodSetToOrderValidityPeriod() {
+        void expectGeneratedMedicationStatementWhenActiveHasEffectivePeriodSetToOrderValidityPeriod() {
```

Comment:
Ha, I think the names have got got a bit confused. The 'When Active' part feels very jarring. If it's a precondition of the expectation, have it at the start?

`WhenSomethingExpectSomething`

Reply:
This has been fixed